### PR TITLE
Silence a warning in the code so it builds with Protobuf 3. Closes #1110

### LINKED ISSUE #1110: protobuf 3 warnings
Hi,

Someone was "nice" enough to upload protobuf 3 into Debian unstable last night. The result is that ola now produces warnings related to protobuf:

    libtool: compile:  g++ -DHAVE_CONFIG_H -I. -Wdate-time -D_FORTIFY_SOURCE=2 -I./include -I./include -Wall -Wformat -W -fvisibility-inlines-hidden -pthread -Werror -DPID_DATA_DIR=\"/usr/share/ola/pids\" -Wno-error=deprecated-declarations -Wno-error=unused-parameter -pthread -c common/rdm/Pids.pb.cc  -fPIC -DPIC -o common/rdm/.libs/common_libolacommon_la-Pids.pb.o
    common/rdm/Pids.pb.cc: In member function 'virtual google::protobuf::uint8* ola::rdm::pid::LabeledValue::InternalSerializeWithCachedSizesToArray(bool, google::protobuf::uint8*) const':
    common/rdm/Pids.pb.cc:517:10: warning: unused parameter 'deterministic' [-Wunused-parameter]
         bool deterministic, ::google::protobuf::uint8* target) const {
              ^~~~~~~~~~~~~
    common/rdm/Pids.pb.cc: In member function 'virtual google::protobuf::uint8* ola::rdm::pid::Range::InternalSerializeWithCachedSizesToArray(bool, google::protobuf::uint8*) const':
    common/rdm/Pids.pb.cc:928:10: warning: unused parameter 'deterministic' [-Wunused-parameter]
         bool deterministic, ::google::protobuf::uint8* target) const {
              ^~~~~~~~~~~~~
    [...]

etc, it repeats a few times.

A workaround is to use `-Wno-error=unused-parameter`.

See also [the Debian bugreport](https://bugs.debian.org/835433)

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

**Comment by yoe** [comment]
Doesn't seem to be enough; the problem repeats in `plugins/artnet/messages/ArtNetConfigMessages.pb.cc`

**Comment by peternewman** [comment]
You didn't tell me about those! :) I've fixed ArtNet, E1.31 and USB Pro plugin protoc issues now.

File: Makefile.am [hunk 1]
```diff
@@ -23,6 +23,11 @@ AM_DISTCHECK_CONFIGURE_FLAGS = --enable-python-libs
 # The common C++ flags. This may include -Werror if fatal-warning was enabled
 # in ./configure. AM_CXXFLAGS defaults to this, and it should be preferred.
 #
+# COMMON_PROTOBUF_CXXFLAGS
+# The common C++ flags with some warnings protobuf code throws disabled. Use
+# this for 3rd party protobuf code that does not build correctly with
```
Comment (nomis52) [inline Makefile.am]: s/3rd party/generated/

**Comment by nomis52** [comment]
LGTM. Just the one comment.

File: Makefile.am (hunk 1 above)
Comment (peternewman) [inline Makefile.am]: Done.

**Comment by peternewman** [comment]
It's in your domain now then @yoe ...

On which note, if you accept the invite to join OLA contributors, we can assign it to you too.

**Comment by peternewman** [comment]
Did this do the job then @yoe ? Can we merge it?

**Comment by yoe** [comment]
wfm

**Comment by peternewman** [comment]
Great thanks. Merging then as @yoe is happy and I fixed @nomis52 comment.

**Pull request merged**